pom.xml

```diff
@@ -147,7 +150,7 @@
             <dependency>
                 <groupId>ru.yandex.qatools.properties</groupId>
                 <artifactId>properties-loader</artifactId>
-                <version>1.5</version>
+                <version>1.6</version>
```

Comment:
this version is deprecated. We will migrate on this one - https://github.com/qatools/properties after stable release